.github/workflows/main.yml

```diff
@@ -20,13 +20,13 @@ jobs:
           main-branch-name: 'master'
       - uses: actions/setup-node@v2
         with:
-          node-version: '14'
+          node-version: '16'
           cache: yarn
       - run: yarn install --frozen-lockfile --non-interactive --no-progress --prefer-offline
       - run: npx nx affected --target=lint --parallel --max-parallel=3
-      - run: npx nx affected --target=build --parallel --max-parallel=3
-      - run: npx nx affected --target=test --parallel --max-parallel=2 --code-coverage
-      - run: npx nx affected --target=e2e --parallel --max-parallel=2
+      - run: npx nx affected --target=build --parallel --max-parallel=3 --exclude=workspace
+      - run: npx nx affected --target=test --parallel --max-parallel=2 --code-coverage --exclude=workspace
+      - run: npx nx affected --target=e2e --parallel --max-parallel=2 --exclude=workspace
```

Comment:
ℹ️ Since Nx 14, root project.json is accounted we can safely exclude it as we use it only to version the workspace